### PR TITLE
add ts definitions

### PR DESCRIPTION
**Pull request by sgtoj**
TypeScript support as described in #22.

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export declare function escapeId(val: string | string[], forbidQualified?: boolean): string;
```
Comment (dougwilson) [inline index.d.ts]: I think `val` should be of type `any` here, right?

File: index.d.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+export declare function escapeId(val: string | string[], forbidQualified?: boolean): string;
+export declare function escape(val: any, stringifyObjects?: boolean, timeZone?: string): string;
+export declare function arrayToList(array: any[], timeZone?: string): string;
+export declare function format(sql: string, values?: any | any[], stringifyObjects?: boolean, timeZone?: string): string;
+export declare function dateToString(date: Date | string, timeZone?: string): string;
```
Comment (dougwilson) [inline index.d.ts]: The `date` arg can be a `number` as well.

Comment (sgtoj) [reply index.d.ts]: Okay. I will update.

File: index.d.ts (hunk 1 above)
Comment (sgtoj) [reply index.d.ts]: I am take another look at the possible paths.

**Comment by dougwilson** [comment]
@sgtoj are you ever going to update? Since this PR new method was added, so please rebase your PR and include that as well.

**Comment by danger89** [comment]
@sgtoj Any update on this?

**Comment by dougwilson** [comment]
Closing due to no response.

**Pull request closed without merging**